commands/displayers/order.go

```diff
@@ -0,0 +1,92 @@
+package displayers
+
+import "github.com/VictorAvelar/mollie-api-go/v3/mollie"
+
+// MollieOrderList wrapper for displaying.
+type MollieOrderList struct {
+	*mollie.OrderList
+}
+
+// KV is a displayable group of key value.
+func (mpl *MollieOrderList) KV() []map[string]interface{} {
+	out := outPrealloc(len(mpl.Embedded.Orders))
```

Comment:
I see the usefulness of this method, however we need 2 things:
1. Create an issue to extend it's usage to other commands
2. Write tests for it.